### ISSUE TITLE
notify() is called when notifications are not configured

### ISSUE THREAD

**Issue by theruleof4** [digitalec/deemon#7]
I just updated recently and now not working anymore, show me this message.

```
Error while sending mail: please run connect() first
Traceback (most recent call last):
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Users\valbe\AppData\Local\Programs\Python\Python37\Scripts\deemon.exe\__main__.py", line 7, in <module>
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\__main__.py", line 5, in main
    Deemon().main()
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\app\main.py", line 187, in main
    self.download_queue(self.queue_list)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\app\main.py", line 136, in download_queue
    self.notify.notify(notify_releases)
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\site-packages\deemon\app\notify.py", line 40, in notify
    self.logger.error("Error while sending mail: " + str(e))
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\smtplib.py", line 284, in __exit__
    self.close()
  File "c:\users\valbe\appdata\local\programs\python\python37\lib\smtplib.py", line 977, in close
    sock = self.sock
AttributeError: 'SMTP_SSL' object has no attribute 'sock'
```

I've checked the logs, seems it keep trying to send an email and needs to be configurated, but I don't wanna send any email, just download

**Comment by digitalec** [comment]
Confirmed. I'll look into this and post back here shortly.

**Comment by digitalec** [comment]
Resolved in v0.4.1

**Comment by theruleof4** [comment]
Worked thank you
